DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadBasicConnectorBinding.cs

```diff
@@ -0,0 +1,137 @@
+using System.Reflection;
+using Autodesk.AutoCAD.DatabaseServices;
+using Sentry.Reflection;
+using Speckle.Connectors.Autocad.HostApp;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models;
+using Speckle.Connectors.DUI.Models.Card;
+using Speckle.Core.Credentials;
+using Speckle.Connectors.Autocad.HostApp.Extensions;
+using Speckle.Connectors.Autocad.Utils;
+
+namespace Speckle.Connectors.Autocad.Bindings;
+
+public class AutocadBasicConnectorBinding : IBasicConnectorBinding
+{
+  public string Name { get; set; } = "baseBinding";
+  public IBridge Parent { get; }
+
+  private readonly DocumentModelStore _store;
+  private readonly AutocadSettings _settings;
+
+  public BasicConnectorBindingCommands Commands { get; }
+
+  private static Document Doc => Application.DocumentManager.MdiActiveDocument;
+
+  public AutocadBasicConnectorBinding(DocumentModelStore store, AutocadSettings settings, IBridge parent)
+  {
+    _store = store;
+    _settings = settings;
+    Parent = parent;
+    Commands = new BasicConnectorBindingCommands(parent);
+    _store.DocumentChanged += (_, _) =>
+    {
+      Commands.NotifyDocumentChanged();
+    };
+  }
+
+  public string GetConnectorVersion() => Assembly.GetAssembly(GetType()).GetNameAndVersion().Version;
+
+  public string GetSourceApplicationName() => _settings.HostAppInfo.Slug;
+
+  public string GetSourceApplicationVersion() => _settings.HostAppVersion.ToString();
+
+  public Account[] GetAccounts() => AccountManager.GetAccounts().ToArray();
+
+  public DocumentInfo GetDocumentInfo()
```

Comment:
When Doc is being injected in the context, it would be good to decide whether this should stay here or become IAutocadDocumentInfo interface, depending on where it is used.